PrivateHeaders/MobileCoreServices/LSApplicationWorkspace.h

```diff
@@ -0,0 +1,143 @@
+//
```

Comment:
Please add the standard facebook copyright header

Reply:
I am happy to do so if you prefer, but are you sure about adding a Facebook copyright to code that belongs to Apple?

Reply:
then just remove the whole;e comment and only keep the `generated` note

Reply:
Done